src/graphql/thegraph/AllV3TicksQuery.ts

```diff
@@ -21,33 +16,30 @@ const query = graphql`
   }
 `
 
-export type Ticks = AllV3TicksQuery$data['ticks']
+export type Ticks = AllV3TicksQuery['ticks']
 export type TickData = Ticks[number]
 
 export default function useAllV3TicksQuery(poolAddress: string | undefined, skip: number, interval: number) {
-  const [data, setData] = useState<AllV3TicksQuery$data | null>(null)
-  const [error, setError] = useState<any>(null)
-  const [isLoading, setIsLoading] = useState(true)
-  const chainId = useAppSelector((state) => state.application.chainId)
+  const {
+    data,
+    loading: isLoading,
+    error,
+  } = useQuery(query, {
+    variables: {
+      poolAddress: poolAddress?.toLowerCase(),
+      skip,
+    },
+    pollInterval: interval,
+    client: apolloClient,
```

Comment:
longer term, is there a way to share the client?

Reply:
I think there is a way but it seems like the long term solution will be using our own endpoint which handles different chains. So I don't want to over invest in a more elegant solution if it would be thrown away https://uniswapteam.slack.com/archives/C02T729PMQE/p1671469080701009